rest-api-guidelines/quality/evolution/naming-conventions.md

```diff
@@ -13,6 +13,14 @@ Every identifier **MUST** be in American English and written in `lowercase`. An
 
 Every URI **MUST** follow the General Rules except for the `camelCase` rule. Instead, a hyphen \(`-`\) **SHOULD** be used to delimit combined words \(kebab-case\). Besides, a URI **MUST NOT** end with a trailing slash \(`/`\).
 
+Plural nouns SHOULD be used in the URI where appropriate to identify collections of data resources (e.g. ``/orders``, ``/products``).
```

Comment:
minor: formatting – it seems that you have double backticks there - ` `` ` is it intentional or should it be just one?

Reply:
it should be just `code`, might be an typo